modules/member/main.tf

```diff
@@ -7,7 +7,26 @@
 
 # Create GuardDuty detector for the member account
 resource "aws_guardduty_detector" "this" {
-  enable = var.enable
+  enable                       = var.enable
+  finding_publishing_frequency = var.finding_publishing_frequency
```

Comment:
Since this module _exclusively_ is about member configs, this argument is not relevant. The value can _only_ be set when configuring a standalone or guardduty administrator account.

Reply:
I removed it.